### PR TITLE
Added Bootstrap writeup

### PR DESCRIPTION
**Pull request by xadahiya**

### CODE REVIEW

**Review by mbad0la — CHANGES_REQUESTED** [review]
Hi @xadahiya 
The content looks fine, although it is quite grammatically incorrect at places. Can you please correct it?

**Comment by xadahiya** [comment]
Sure, will do

**Comment by xadahiya** [comment]
@mbad0la can you point what errors are you talking about?

File: Frameworks.md [hunk 1]
```diff
@@ -71,3 +71,31 @@ Meteor by default uses Blaze templating system that resembles Handlebars, and is
   * https://github.com/meteorhacks/npm
   * https://github.com/alanning/meteor-roles
   * https://github.com/arunoda/meteor-up/tree/mupx
+  
+### [Bootstrap](http://getbootstrap.com)
+
+Bootstrap is an open-source Javascript framework developed by the team at Twitter. It is a combination of HTML, CSS, and Javascript code designed to help build user interface components. Bootstrap was also programmed to support both HTML5 and CSS3.
```
Comment (mbad0la) [inline Frameworks.md:77]: Bootstrap isn't a JavaScript Framework

File: Frameworks.md [hunk 2]
```diff
@@ -71,3 +71,31 @@ Meteor by default uses Blaze templating system that resembles Handlebars, and is
...
+
+Also it is called Front-end-framework.
```
Comment (mbad0la) [inline Frameworks.md:79]: Improper sentence

File: Frameworks.md [hunk 3]
```diff
@@ -71,3 +71,31 @@ Meteor by default uses Blaze templating system that resembles Handlebars, and is
...
+
+Bootstrap is a free collection of tools for creating a websites and web applications.
```
Comment (mbad0la) [inline Frameworks.md:81]: `a websites` is wrong

File: Frameworks.md [hunk 4]
```diff
@@ -71,3 +71,31 @@ Meteor by default uses Blaze templating system that resembles Handlebars, and is
...
+
+It contains HTML and CSS-based design templates for typography, forms, buttons, navigation and other interface components, as well as optional JavaScript extensions.
+
+#### Some Reasons for programmers preferred Bootstrap Framework
```
Comment (mbad0la) [inline Frameworks.md:85]: Improper sentence

**Review by mbad0la — COMMENTED** [review]
I guess this is fair enough to give you a rough idea.

**Comment by xadahiya** [comment]
Sorry I took it from stack overflow, will fix this right away :-) .

**Comment by mbad0la** [comment]
@xadahiya please rebase your branch as well. Thanks!

**Comment by mbad0la** [comment]
@xadahiya please let me know the progress, else I'll be closing this PR in a day or two.

**Pull request closed without merging**